refactor: use the new cas server (#128)
jina-ai/dalle-flow: README.md

You can modify and extend the server flow as you like, e.g. changing the model, adding persistence, or even auto-posting to Instagram/OpenSea. With Jina and DocArray, you can easily make DALL·E Flow [cloud-native and ready for production](https://github.com/jina-ai/jina). 

### Use the CLIP-as-service

> To reduce the usage of vRAM, you can use the `CLIP-as-service` as external executor freely hosted by `grpcs://api.clip.jina.ai:2096`.  
> First, make sure you have created access token from [console website](https://console.clip.jina.ai/get_started), or CLI as described

Reviewer: ```suggestion
To reduce the usage of vRAM, you can use the `CLIP-as-service` as an external executor freely available at `grpcs://api.clip.jina.ai:2096`.  
First, make sure you have created an access token from [console website](https://console.clip.jina.ai/get_started), or CLI as following
```

```bash
jina auth token create <name of PAT> -e <expiration days>
```

Then, you need to change the executor related configs (`external`, `tls` and `grpc_metadata`) from [`flow.yml`](./flow.yml).

```yaml
...
  - name: clip_encoder

>     uses: jinahub+docker://CLIPTorchEncoder/latest-gpu

Reviewer: I'm thinking why we need this info since the service is not managed by users.

Author: Let's keep it here to tell the user what's the exactly executor in this flow.

    host: 'api.clip.jina.ai'
    port: 2096
    tls: true
    external: true
    grpc_metadata:
      authorization: "<your access token>"
    needs: [gateway]
...
  - name: rerank
    uses: jinahub+docker://CLIPTorchEncoder/latest-gpu

>     host: 'demo-cas.jina.ai'

Reviewer: ```suggestion
    host: 'api.clip.jina.ai'
```
